packages/lodestar/src/network/peers/client.ts

```diff
@@ -0,0 +1,61 @@
+export enum ClientKind {
+  Lighthouse = "Lighthouse",
+  Nimbus = "Nimbus",
+  Teku = "Teku",
+  Prysm = "Prysm",
+  Lodestar = "Lodestar",
+  Unknown = "Unknown",
+}
+
+export type Client = {
+  kind: ClientKind;
+  version: string;
+  osVersion: string;
+};
+
+export function clientFromAgentVersion(agentVersion: string): Client {
+  const agentSplit = agentVersion.split("/");
```

Comment:
split has to manipulate the string more than necessary. If we only care about the ClientKind split the string once with
```ts
const slashIndex = agentVersion.indexOf("/")
const agentKind = slashIndex >= 0 ? agentVersion.slice(0, slashIndex) : null
```
Then return `ClientKind`